e2e/autoscript-suites/README.md

```diff
@@ -0,0 +1,58 @@
+# Autoscript Suites
+
+This is a package that contains [playwright](https://playwright.dev/)
+tests for the [autoscript-apps](../autoscript-apps) application.
+
+## How to write a test
+
+In order to write a test navigate to the [src](./src) directory.
+
+Once there you will see a folder called suites.
+In here we can write a test which follows the name of the
+feature we are testing, and a .test.ts suffix.
+
+`my-auth.test.ts` as an example.
+
+Next, we typically use `playwrights` `test` blocks and leverage a
```

Comment:
Rather than this:

`playwrights`

... I'd recommend this:

Playwright's

The former feels odd, since it's plural, rather than possesive.